DEVELOPMENT.md

```diff
@@ -26,89 +30,65 @@ brew install go # MacOS
 sudo apt-get install golang # Linux
 ```
 
-### 2. Verification of Go Installation
+After installing Go, download the `goimports` package:
 
-Confirm your Go installation via this command:
-
-```bash
-go version
 ```
-
-### 3. Project Execution
-
-Reach the project directory and perform these commands:
-
-```bash
-cd path/to/project
-go mod download # download dependencies
-go run .  # execute the program
+go install golang.org/x/tools/cmd/goimports@latest
 ```
 
-This initiates a server on `port 8080`. You should see:
-
-```bash
-[GIN-debug] Listening and serving HTTP on :8080
-```
+### Verification of Go installation
 
-Interact with the server by sending HTTP requests:
+Confirm your Go installation via this command:
 
 ```bash
-curl localhost:8080
+go version
 ```
 
```

Comment:
part 2 of 2 - add:
```markdown
### Docker Installation and Usage (if absent)
If Docker is not installed on your system, you can download it by visiting [Docker's official site](https://docs.docker.com/). Docker can significantly simplify the setup process and ensure consistent running environments across different systems.
```

Reply:
#### Great job on the recent documentation updates!  @mattkuznicki-ll 👌
#### quick review:
1. **Provided detailed instructions for setup and testing.**
2. **Promoted good practices for high-quality code.**
3. **Added 'Pull Requests' and 'Troubleshooting' for collaboration.**
4. **Set clear guidelines for effective pull requests.**
5. **Included troubleshooting steps to save time.**

####  note: recommended enhancement (non-stopper to mergue): + a link to Docker installation guide for contributors without Docker setup.